index.css

```diff
@@ -0,0 +1,120 @@
+@import url(/styles/reset.css);
+@import url(/styles/product-page.css);
+
+
+
+.container{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    margin: 12rem;
+}
+.small-space{
+    margin-bottom: 1rem;
+}
+
+.content-area{
+    display: flex;
+}
+.hero-image .hero-img{
+    position: relative;
+}
+.content-heading .hero-heading{
+    position: absolute;
+    left:10%;
+    bottom: 43%;
+    color: var(--secondary-color);
+}
+.card-component-area{
+    margin: 2rem;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+}
+.basic-card-container.horizontal{
+    display: flex;
+    background-color: var(--secondary-color);
+    padding: 1rem;
+    border-radius: 0.3rem;
+    margin: 1rem;
+    max-width: 60rem;
+    box-shadow: 1px 1px 15px 0 var(--tertiary-color);
+}
+.card-photo.horizontal{
+    width:auto;
+    height: auto;
+    margin: 2rem;
+    max-width: 18rem;
+    max-height: 20rem;
+    border-radius: 0.3rem;
+}
+.photo-area.horizontal{
+    min-width: 15rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+.price{
+    display: flex;
+}
+.card-subheading.detail{
+    margin-top: 1rem;
+    font-size: 1rem;
+}
+.card-heading{
+    font-size: 2rem;
+    margin: 5rem 0 0 0.5rem;
+}
+.card-subheading{
+    font-size: 1.5rem;
+    margin: 1rem 0 0.5rem 0.5rem;
+}
+.card-body{
+    margin: 0.5rem;
+}
+.card-footer.horizontal{
+    margin-top: 2rem;
+    display: flex;
```

Comment:
You can create utility classes for different flex as it is reused may times.

Reply:
Okay Ill just add it to my reset.css Thanks for the suggestion